compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmDesugarPhase.java

```diff
@@ -160,275 +154,119 @@ private static void rewriteRecordInitFunction(BIRFunction func, BRecordType reco
         func.localVars = updatedLocalVars;
     }
 
-    private static BIRFunctionParameter getFunctionParam(BIRFunctionParameter localVar) {
-        if (localVar == null) {
-            throw new BLangCompilerException("Invalid function parameter");
-        }
-
-        return localVar;
+    static void encodeModuleIdentifiers(BIRNode.BIRPackage module) {
```

Comment:
Is an access modifier not necessary here?

Reply:
It was previously there. Hence, that is okay kept it.